src/components/SlideShow/SlideShow.tsx

```diff
@@ -76,16 +76,28 @@ export const SlideShow = ({
   };
   const increment = () => setIndex(normalize(index + 1));
   const decrement = () => setIndex(normalize(index - 1));
-
+  const setAuto = () =>
+    setInterval(() => {
+      setIndex(index => normalize(index+1));
```

Comment:
https://stackoverflow.com/questions/53395147/use-react-hook-to-implement-a-self-increment-counter